tests/rptest/wasm/templates/identity_transform.j2

```diff
@@ -1,6 +1,7 @@
 const {
   SimpleTransform,
   PolicyError,
+  PolicyInjection
 } = require("@vectorizedio/wasm-api");
 const transform = new SimpleTransform();
 transform.subscribe([{{input_topics}}]);
```

Comment:
put the PolicyInjection value here

Reply:
This is a template, so the `{{input_topics}}` string will be evaluated to contain the correct topic/value pairs